addons/account/models/account_journal_dashboard.py

```diff
@@ -102,6 +104,33 @@ def get_line_graph_datas(self):
         color = '#875A7B' if '+e' in version else '#7c7bad'
         return [{'values': data, 'title': graph_title, 'key': graph_key, 'area': True, 'color': color}]
 
+    # Below method is used to get data of bank statemens
+    @api.multi
+    def get_bank_stmt_line_graph_datas(self):
```

Comment:
After suppressing the previous diff, rename this method to `get_line_graph_datas` and remove the old method.